db/migrate/20160807000122_remove_queue_from_email_digest_agent_memory.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveQueueFromEmailDigestAgentMemory < ActiveRecord::Migration
+  def up
+    Agents::EmailDigestAgent.all.each do |agent|
+      agent.memory.delete("queue")
+      agent.save!
```

Comment:
`validate: false` might be wise in a migration where the Agent might be invalid.